doc/source/whatsnew/v0.18.1.txt

```diff
@@ -656,3 +656,4 @@ Bug Fixes
 - Bug in ``to_numeric`` with ``Index`` returns ``np.ndarray``, rather than ``Index`` (:issue:`12777`)
 - Bug in ``to_numeric`` with datetime-like may raise ``TypeError`` (:issue:`12777`)
 - Bug in ``to_numeric`` with scalar raises ``ValueError`` (:issue:`12777`)
+- Bug where ``_parse_cell`` raised OverflowError if numeric value was placed in excel cell of type 'date'  (:issue:`10001`)
```

Comment:
move to 0.18.2 (just about to release 0.18.1). make this a more meaningful message to the user.